lib/algos/list/delimiter.js

```diff
@@ -56,6 +62,8 @@ class Delimiter extends Extension {
         this.prefix = parameters.prefix;
         this.marker = parameters.marker;
         this.maxKeys = parameters.maxKeys || 1000;
+        this.alphabeticalOrder = parameters.alphabeticalOrder || true;
```

Comment:
won't this always be true since if parameters.alphaticalOrder is false or undefined this will return true?